.github/workflows/ci.yml

```diff
@@ -0,0 +1,49 @@
+name: CI
+
+on:
+    push:
+    workflow_dispatch:
+
+jobs:
+    build:
+        runs-on: ubuntu-latest
+        permissions:
+            contents: read
+            packages: write
+
+        steps:
+            - name: Checkout the Repository
+              uses: actions/checkout@v3
+
+            - name: Set up Docker Buildx
+              id: buildx
+              uses: docker/setup-buildx-action@v2
+
+            - name: Login to Github Container Registry
+              uses: docker/login-action@v2
+              with:
+                  registry: ghcr.io
+                  username: ${{ github.actor }}
+                  password: ${{ secrets.GITHUB_TOKEN }}
+
+            - name: Prepare production environment variables
+              shell: bash
+              env:
+                  ENV_PRODUCTION: ${{ secrets.ENV_PRODUCTION }}
```

Comment:
We need to add the env value for the backend URL in here I think. 
In our case, it should be the one in the .envsample file.

Reply:
Sure